workflow/usage.rst

```diff
@@ -501,6 +501,48 @@ This class has two more methods:
 :method:`Symfony\\Component\\Workflow\\Event\\GuardEvent::setBlocked`
     Sets the blocked value.
 
+.. _workflow-usage-using-guards:
+
+Using Guards
+------------
+
+The component has a guard logic to control the execution of your workflow on top of your configuration.
+
+It allows you to execute your custom logic to decide if the transition is blocked or not, before actually
+applying this transition.
+
+You have multiple optional ways to use guards in your workflow.
+
+The first way is :ref:`with the guard event <workflow-usage-guard-events>`, which allows you to implement
+any desired feature.
+
+Another one is via the configuration and its specific entry `guard` on a transition.
```

Comment:
I know it's tricky but, in RST format, the `<code>` elements are different than in Markdown. Instead of a single backtick, you need to add two:

```
`guard` -> ``guard``
```

Reply:
Understood